src/lib/Err.hs

```diff
@@ -110,15 +203,53 @@ layout :: LayoutOptions
 layout = if unbounded then LayoutOptions Unbounded else defaultLayoutOptions
   where unbounded = unsafePerformIO $ (Just "1"==) <$> lookupEnv "DEX_PPRINT_UNBOUNDED"
 
+traverseMergingErrs :: (Traversable f, FallibleApplicative m)
+                    => (a -> m b) -> f a -> m (f b)
+traverseMergingErrs f xs =
+  fromSeqFallibleApplicative $ traverse (\x -> SeqFallibleApplicative $ f x) xs
```

Comment:
nit: this thing under traverse is just `SeqFaillableApplicative . f`

Reply:
and if you want to avoid typing `fromSeqFaillableApplicative`, `coerce` would work too (but might be a little less direct)